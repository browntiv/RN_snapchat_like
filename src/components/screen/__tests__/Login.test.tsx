import 'react-native';
import * as React from 'react';
import login from '../Screen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import { shallow, render } from 'enzyme';

describe('rendering test', () => {
  const wrapper = shallow(
    <login />,
  );

  it('renders as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
