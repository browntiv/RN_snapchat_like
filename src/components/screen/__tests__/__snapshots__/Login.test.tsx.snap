// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering test renders as expected 1`] = `<login />`;
